GameVault.Server/Controllers/AuthController.cs

```diff
@@ -189,6 +191,12 @@ public async Task<ActionResult<DataResponse<string>>> RegisterVendor([FromBody]
 
             if (id is not null)
             {
+                var adminUsers = await _firestore.QueryDocumentsAsyncWithId<User>("users", "Type", 0);
```

Comment:
```suggestion
                var adminUsers = await _firestore.QueryDocumentsAsyncWithId<User>("users", "Type", (int)AccountType.Admin);
```
For clarity, could use ^